notebooks/notebooks/iclr2022/win-rate-vs-training-steps.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            df19[["victim_name_v2", "adv_steps"]].groupby("victim_name_v2").min().adv_steps
</pre><p><code>groupby</code> returns an object which you can call multiple times -- so can put the groupby on its own line assigned to a temporary variable <code>grp</code>  say and then call <code>grp.min().adv_steps</code> and <code>grp.max().adv_steps</code>.I think you might also be able to do <code>grp.adv_steps.min()</code> and this seems more natural to me (rather than taking the min of everything and then just picking one attribute from it) but not 100% that works.      </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>